### PR TITLE
cloud: add release notes for 20240924

### PR DESCRIPTION
**Pull request by hfxsd**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v8.4 (TiDB 8.4 versions)
- [ ] v8.3 (TiDB 8.3 versions)
- [ ] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by qqqdan — APPROVED** [review]

File: tidb-cloud/tidb-cloud-release-notes.md [hunk 1]
```diff
@@ -8,6 +8,12 @@ aliases: ['/tidbcloud/supported-tidb-versions','/tidbcloud/release-notes']
 
 This page lists the release notes of [TiDB Cloud](https://www.pingcap.com/tidb-cloud/) in 2024.
 
+## September 24, 2024
+
+**General changes**
+
+- Provide a new [TiFlash vCPU and RAM](/tidb-cloud/size-your-cluster.md#tiflash-vcpu-and-ram) for [TiDB Cloud Dedicated](/tidb-cloud/select-cluster-tier.md#tidb-cloud-dedicated) clusters hosted on AWS: `32 vCPU, 128 GiB`
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-release-notes.md:15]:
```suggestion
- Provide a new [TiFlash vCPU and RAM size](/tidb-cloud/size-your-cluster.md#tiflash-vcpu-and-ram) for [TiDB Cloud Dedicated](/tidb-cloud/select-cluster-tier.md#tidb-cloud-dedicated) clusters hosted on AWS: `32 vCPU, 128 GiB`
```

**Review by qiancai — APPROVED** [review]
Rest LGTM

**Review by zhangyangyu — APPROVED** [review]

File: tidb-cloud/tidb-cloud-release-notes.md [hunk 2]
```diff
@@ -8,6 +8,39 @@ aliases: ['/tidbcloud/supported-tidb-versions','/tidbcloud/release-notes']
...
+
+  TiDB Cloud CLI enhances the following features:
+
+    - Improve debug logging. Now it can log credentials and user-agent.
+    - Speed up local export file downloads from tens of KiB per second to tens of MiB per second.
+
+  TiDB Cloud CLI replaces or removes the following features:
+
+    - The `--s3.bucket-uri` flag is replaced by `-s3.uri` in `ticloud serverless export create`.
+    - `--database` and `--table` flags are removed in ticloud serverless export create. Instead, you can use `--sql`, `--where`, and `--filter` flags.
```
Comment (shiyuhang0) [inline tidb-cloud/tidb-cloud-release-notes.md:41]:
```suggestion
    - `--database` and `--table` flags are removed in `ticloud serverless export create`. Instead, you can use `--sql`, `--where`, and `--filter` flags.
```

File: tidb-cloud/tidb-cloud-release-notes.md [hunk 3]
```diff
@@ -8,6 +8,39 @@ aliases: ['/tidbcloud/supported-tidb-versions','/tidbcloud/release-notes']
...
+    - Support creating a branch from a branch and specific timestamp.
+
+  TiDB Cloud CLI enhances the following features:
+
+    - Improve debug logging. Now it can log credentials and user-agent.
+    - Speed up local export file downloads from tens of KiB per second to tens of MiB per second.
+
+  TiDB Cloud CLI replaces or removes the following features:
+
+    - The `--s3.bucket-uri` flag is replaced by `-s3.uri` in `ticloud serverless export create`.
```
Comment (shiyuhang0) [inline tidb-cloud/tidb-cloud-release-notes.md:40]:
```suggestion
    - The `--s3.bucket-uri` flag is replaced by `--s3.uri` in `ticloud serverless export create`.
```

File: tidb-cloud/tidb-cloud-release-notes.md [hunk 4]
```diff
@@ -8,6 +8,39 @@ aliases: ['/tidbcloud/supported-tidb-versions','/tidbcloud/release-notes']
...
+
+
+**CLI changes**
+
+- Release [TiDB Cloud CLI v1.0.0-beta.2](https://github.com/tidbcloud/tidbcloud-cli/releases/tag/v1.0.0-beta.2).
+
+    TiDB Cloud CLI provides the following new features:
+
+    - Support SQL user management for [TiDB Cloud Serverless](/tidb-cloud/select-cluster-tier.md#tidb-cloud-serverless) clusters via [`ticloud serverless sql-user`](/tidb-cloud/ticloud-serverless-sql-user-create.md).
+    - Allow disabling the public endpoint for [TiDB Cloud Serverless](/tidb-cloud/select-cluster-tier.md#tidb-cloud-serverless) clusters in `ticloud serverless create` and `ticloud serverless update`.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-release-notes.md:25]:
```suggestion
    - Allow disabling the public endpoint for [TiDB Cloud Serverless](/tidb-cloud/select-cluster-tier.md#tidb-cloud-serverless) clusters in [`ticloud serverless create`](/tidb-cloud/ticloud-cluster-create.md) and [`ticloud serverless update`](/tidb-cloud/ticloud-serverless-update.md).
```

File: tidb-cloud/tidb-cloud-release-notes.md [hunk 5]
```diff
@@ -8,6 +8,39 @@ aliases: ['/tidbcloud/supported-tidb-versions','/tidbcloud/release-notes']
...
+    - Add the [`ticloud auth whoami`](/tidb-cloud/ticloud-auth-whoami.md) command to get information about the current user when using OAuth authentication.
+    - Support `--sql`, `--where`, and `--filter` flags in `ticloud serverless export create` to choose source tables flexibly.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-release-notes.md:27]:
```suggestion
    - Support `--sql`, `--where`, and `--filter` flags in [`ticloud serverless export create`](/tidb-cloud/ticloud-serverless-export-create.md) to choose source tables flexibly.
```

File: tidb-cloud/tidb-cloud-release-notes.md [hunk 6]
```diff
@@ -8,6 +8,39 @@ aliases: ['/tidbcloud/supported-tidb-versions','/tidbcloud/release-notes']
...
+
+  TiDB Cloud CLI enhances the following features:
+
+    - Improve debug logging. Now it can log credentials and user-agent.
+    - Speed up local export file downloads from tens of KiB per second to tens of MiB per second.
+
+  TiDB Cloud CLI replaces or removes the following features:
+
+    - The `--s3.bucket-uri` flag is replaced by `--s3.uri` in `ticloud serverless export create`.
+    - `--database` and `--table` flags are removed in `ticloud serverless export create`. Instead, you can use `--sql`, `--where`, and `--filter` flags.
```
Comment (shiyuhang0) [inline tidb-cloud/tidb-cloud-release-notes.md:41]:
```suggestion
    - The `--database` and `--table` flags are removed in `ticloud serverless export create`. Instead, you can use `--sql`, `--where`, and `--filter` flags.
```

File: tidb-cloud/tidb-cloud-release-notes.md [hunk 7]
```diff
@@ -8,6 +8,39 @@ aliases: ['/tidbcloud/supported-tidb-versions','/tidbcloud/release-notes']
...
+  TiDB Cloud CLI enhances the following features:
+
+    - Improve debug logging. Now it can log credentials and user-agent.
+    - Speed up local export file downloads from tens of KiB per second to tens of MiB per second.
+
+  TiDB Cloud CLI replaces or removes the following features:
+
+    - The `--s3.bucket-uri` flag is replaced by `--s3.uri` in `ticloud serverless export create`.
+    - The `--database` and `--table` flags are removed in `ticloud serverless export create`. Instead, you can use `--sql`, `--where`, and `--filter` flags.
+    - `ticloud serverless update` cannot update the annotations field anymore.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-release-notes.md:42]:
```suggestion
    - The `--s3.bucket-uri` flag is replaced by `--s3.uri` in [`ticloud serverless export create`](/tidb-cloud/ticloud-serverless-export-create.md).
    - The `--database` and `--table` flags are removed in [`ticloud serverless export create`](/tidb-cloud/ticloud-serverless-export-create.md). Instead, you can use `--sql`, `--where`, and `--filter` flags.
    - [`ticloud serverless update`](/tidb-cloud/ticloud-serverless-update.md) cannot update the annotations field anymore.
```

File: tidb-cloud/tidb-cloud-release-notes.md [hunk 8]
```diff
@@ -8,6 +8,39 @@ aliases: ['/tidbcloud/supported-tidb-versions','/tidbcloud/release-notes']
...
+    TiDB Cloud CLI provides the following new features:
+
+    - Support SQL user management for [TiDB Cloud Serverless](/tidb-cloud/select-cluster-tier.md#tidb-cloud-serverless) clusters via [`ticloud serverless sql-user`](/tidb-cloud/ticloud-serverless-sql-user-create.md).
+    - Allow disabling the public endpoint for [TiDB Cloud Serverless](/tidb-cloud/select-cluster-tier.md#tidb-cloud-serverless) clusters in `ticloud serverless create` and `ticloud serverless update`.
+    - Add the [`ticloud auth whoami`](/tidb-cloud/ticloud-auth-whoami.md) command to get information about the current user when using OAuth authentication.
+    - Support `--sql`, `--where`, and `--filter` flags in `ticloud serverless export create` to choose source tables flexibly.
+    - Support exporting data to CSV and Parquet files.
+    - Support exporting data to Amazon S3 using role ARN as credentials, to GCS and Azure Blob Storage.
+    - Support importing data from Amazon S3, GCS, and Azure Blob Storage.
+    - Support creating a branch from a branch and specific timestamp.
```
Comment (shiyuhang0) [inline tidb-cloud/tidb-cloud-release-notes.md:31]:
```suggestion
    - Support creating a branch from a branch and a specific timestamp.
```

File: tidb-cloud/tidb-cloud-release-notes.md [hunk 9]
```diff
@@ -8,6 +8,39 @@ aliases: ['/tidbcloud/supported-tidb-versions','/tidbcloud/release-notes']
...
+- Release [TiDB Cloud CLI v1.0.0-beta.2](https://github.com/tidbcloud/tidbcloud-cli/releases/tag/v1.0.0-beta.2).
+
+    TiDB Cloud CLI provides the following new features:
+
+    - Support SQL user management for [TiDB Cloud Serverless](/tidb-cloud/select-cluster-tier.md#tidb-cloud-serverless) clusters via [`ticloud serverless sql-user`](/tidb-cloud/ticloud-serverless-sql-user-create.md).
+    - Allow disabling the public endpoint for [TiDB Cloud Serverless](/tidb-cloud/select-cluster-tier.md#tidb-cloud-serverless) clusters in [`ticloud serverless create`](/tidb-cloud/ticloud-cluster-create.md) and [`ticloud serverless update`](/tidb-cloud/ticloud-serverless-update.md).
+    - Add the [`ticloud auth whoami`](/tidb-cloud/ticloud-auth-whoami.md) command to get information about the current user when using OAuth authentication.
+    - Support `--sql`, `--where`, and `--filter` flags in [`ticloud serverless export create`](/tidb-cloud/ticloud-serverless-export-create.md) to choose source tables flexibly.
+    - Support exporting data to CSV and Parquet files.
+    - Support exporting data to Amazon S3 using role ARN as credentials, to GCS and Azure Blob Storage.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-release-notes.md:29]:
```suggestion
    - Support exporting data to Amazon S3 using role ARN as credentials, to Google Cloud Storage and Azure Blob Storage.
```

**Review by qiancai — APPROVED** [review]
Rest LGTM

File: tidb-cloud/tidb-cloud-release-notes.md [hunk 10]
```diff
@@ -8,6 +8,39 @@ aliases: ['/tidbcloud/supported-tidb-versions','/tidbcloud/release-notes']
...
+
+    TiDB Cloud CLI provides the following new features:
+
+    - Support SQL user management for [TiDB Cloud Serverless](/tidb-cloud/select-cluster-tier.md#tidb-cloud-serverless) clusters via [`ticloud serverless sql-user`](/tidb-cloud/ticloud-serverless-sql-user-create.md).
+    - Allow disabling the public endpoint for [TiDB Cloud Serverless](/tidb-cloud/select-cluster-tier.md#tidb-cloud-serverless) clusters in [`ticloud serverless create`](/tidb-cloud/ticloud-cluster-create.md) and [`ticloud serverless update`](/tidb-cloud/ticloud-serverless-update.md).
+    - Add the [`ticloud auth whoami`](/tidb-cloud/ticloud-auth-whoami.md) command to get information about the current user when using OAuth authentication.
+    - Support `--sql`, `--where`, and `--filter` flags in [`ticloud serverless export create`](/tidb-cloud/ticloud-serverless-export-create.md) to choose source tables flexibly.
+    - Support exporting data to CSV and Parquet files.
+    - Support exporting data to Amazon S3 using role ARN as credentials, to Google Cloud Storage and Azure Blob Storage.
+    - Support importing data from Amazon S3, GCS, and Azure Blob Storage.
```
Comment (hfxsd) [inline tidb-cloud/tidb-cloud-release-notes.md:30]:
```suggestion
    - Support importing data from Amazon S3, Google Cloud Storage, and Azure Blob Storage.
```

Comment (shiyuhang0) [inline tidb-cloud/tidb-cloud-release-notes.md:30]:
```suggestion
    - Support importing data from Amazon S3, Google Cloud Storage, and Azure Blob Storage.
```

File: tidb-cloud/tidb-cloud-release-notes.md [hunk 11]
```diff
@@ -8,6 +8,39 @@ aliases: ['/tidbcloud/supported-tidb-versions','/tidbcloud/release-notes']
...
+- Release [TiDB Cloud CLI v1.0.0-beta.2](https://github.com/tidbcloud/tidbcloud-cli/releases/tag/v1.0.0-beta.2).
+
+    TiDB Cloud CLI provides the following new features:
+
+    - Support SQL user management for [TiDB Cloud Serverless](/tidb-cloud/select-cluster-tier.md#tidb-cloud-serverless) clusters via [`ticloud serverless sql-user`](/tidb-cloud/ticloud-serverless-sql-user-create.md).
+    - Allow disabling the public endpoint for [TiDB Cloud Serverless](/tidb-cloud/select-cluster-tier.md#tidb-cloud-serverless) clusters in `ticloud serverless create` and `ticloud serverless update`.
+    - Add the [`ticloud auth whoami`](/tidb-cloud/ticloud-auth-whoami.md) command to get information about the current user when using OAuth authentication.
+    - Support `--sql`, `--where`, and `--filter` flags in `ticloud serverless export create` to choose source tables flexibly.
+    - Support exporting data to CSV and Parquet files.
+    - Support exporting data to Amazon S3 using role ARN as credentials, to GCS and Azure Blob Storage.
```
Comment (shiyuhang0) [inline tidb-cloud/tidb-cloud-release-notes.md]:
```suggestion
    - Support exporting data to Amazon S3 using role ARN as credentials, and also support exporting to Google Cloud Storage and Azure Blob Storage.
```

File: tidb-cloud/tidb-cloud-release-notes.md [hunk 12]
```diff
@@ -8,6 +8,39 @@ aliases: ['/tidbcloud/supported-tidb-versions','/tidbcloud/release-notes']
...
+- Release [TiDB Cloud CLI v1.0.0-beta.2](https://github.com/tidbcloud/tidbcloud-cli/releases/tag/v1.0.0-beta.2).
+
+    TiDB Cloud CLI provides the following new features:
+
+    - Support SQL user management for [TiDB Cloud Serverless](/tidb-cloud/select-cluster-tier.md#tidb-cloud-serverless) clusters via [`ticloud serverless sql-user`](/tidb-cloud/ticloud-serverless-sql-user-create.md).
+    - Allow disabling the public endpoint for [TiDB Cloud Serverless](/tidb-cloud/select-cluster-tier.md#tidb-cloud-serverless) clusters in [`ticloud serverless create`](/tidb-cloud/ticloud-cluster-create.md) and [`ticloud serverless update`](/tidb-cloud/ticloud-serverless-update.md).
+    - Add the [`ticloud auth whoami`](/tidb-cloud/ticloud-auth-whoami.md) command to get information about the current user when using OAuth authentication.
+    - Support `--sql`, `--where`, and `--filter` flags in [`ticloud serverless export create`](/tidb-cloud/ticloud-serverless-export-create.md) to choose source tables flexibly.
+    - Support exporting data to CSV and Parquet files.
+    - Support exporting data to Amazon S3 using role ARN as credentials, to Google Cloud Storage and Azure Blob Storage.
```
Comment (shiyuhang0) [inline tidb-cloud/tidb-cloud-release-notes.md:29]:
```suggestion
    - Support exporting data to Amazon S3 using role ARN as credentials, and also support exporting to Google Cloud Storage and Azure Blob Storage.
```

**Review by shiyuhang0 — APPROVED** [review]

File: tidb-cloud/tidb-cloud-release-notes.md [hunk 13]
```diff
@@ -8,6 +8,39 @@ aliases: ['/tidbcloud/supported-tidb-versions','/tidbcloud/release-notes']
 
 This page lists the release notes of [TiDB Cloud](https://www.pingcap.com/tidb-cloud/) in 2024.
 
+## September 24, 2024
+
+**General changes**
+
+- Provide a new [TiFlash vCPU and RAM size](/tidb-cloud/size-your-cluster.md#tiflash-vcpu-and-ram) for [TiDB Cloud Dedicated](/tidb-cloud/select-cluster-tier.md#tidb-cloud-dedicated) clusters hosted on AWS: `32 vCPU, 128 GiB`
+
```
Comment (hfxsd) [inline tidb-cloud/tidb-cloud-release-notes.md:16]:
```suggestion
```

**Pull request merged**